wsi/drm/drmdisplay.cpp

```diff
@@ -135,11 +135,13 @@ bool DrmDisplay::ConnectDisplay(const drmModeModeInfo &mode_info,
   SetPowerMode(power_mode_);
 
   // This is a valid case on DSI panels.
-  if (!broadcastrgb_props)
+  if (broadcastrgb_props == NULL)
     return true;
 
-  if (!(broadcastrgb_props->flags & DRM_MODE_PROP_ENUM))
+  if (!(broadcastrgb_props->flags & DRM_MODE_PROP_ENUM)) {
+    drmModeFreeProperty(broadcastrgb_props);
```

Comment:
Good catch but just a minor nit - this is not a NULL pointer dereference issue.  Was this reported by Klockwork?

Reply:
Klocwork caught this as a possible memory leak